src/renderer/api/navidrome/navidrome-types.ts

```diff
@@ -343,6 +343,17 @@ const removeFromPlaylistParameters = z.object({
     id: z.array(z.string()),
 });
 
+const shareItem = z.object({
+    id: z.string(),
+});
+
+const shareItemParameters = z.object({
+    description: z.string(),
+    downloadable: z.boolean(),
```

Comment:
Navidrome (and subsonic) also supports `expires` value.

Reply:
I'll get to work on implementing this right now along with the server version check.